crates/holochain/src/conductor/manager/mod.rs

```diff
@@ -161,7 +217,61 @@ mod test {
         if let Err(_) = send_task_handle.send(handle).await {
             panic!("Failed to send the handle");
         }
-        main_handle.await??;
+        main_handle.await???;
         Ok(())
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    #[should_panic]
```

Comment:
I'm a bit confused that this test relies on panicking, at is looks like we could expect there to be an `Err` and fail otherwise, without propagating the panic to the test runner

Reply:
Yeh my bad I forgot to use the handle_shutdown function but this panics if it gets an error on shutdown because there's nothing else to do at that point.